roles/container_runtime/tasks/package_crio.yml

```diff
@@ -69,6 +69,11 @@
     dest: /etc/sysconfig/crio-network
     src: crio-network.j2
 
+- name: Fixup SELinux permissions for docker
+  shell: |
+          semanage fcontext -a -e /var/lib/docker/overlay2 /var/lib/containers/overlay2
+          restorecon -R -v /var/lib/containers/overlay2
```

Comment:
NIT: The -v is for verbose output.  Not sure it is needed, but it will not hurt anything.